File: modul 3/Modul 3 Bahasa Python/PRAK305-2410817310007-MuhammadNaufalAbdillah.py
Total_detik = int(input("Masukan detik: "))
hari = Total_detik//86400
Total_detik%= 86400
jam = Total_detik// 3600
Total_detik %= 3600
menit = Total_detik//60
Total_detik %= 60

if hari > 0:
    print(f"{hari} hari")
print(f"{jam:02}:{menit:02}:{Total_detik:02}")

File: Modul 3/Modul 3 Bahasa C/PRAK305-2410817310007-MuhammadNaufalAbdillah.c
#include <stdio.h>
int main (){
    int Total_detik;
   
printf("Masukan detik : ");
scanf("%d", &Total_detik);
    int hari = Total_detik/ 86400;
    Total_detik %= 86400;
    int jam = Total_detik/ 3600;
    Total_detik %= 3600;
    int menit = Total_detik/ 60;
    Total_detik %= 60;

if (hari > 0) {printf("%d hari %02d:%02d:%02d\n", hari, jam, menit, Total_detik);}
 else {printf("%02d:%02d:%02d\n", jam, menit, Total_detik);}
return 0;
}